pkg/operator/ceph/controller/spec.go

```diff
@@ -771,20 +769,28 @@ func LogCollectorContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Contai
 		maxLogSize = resource.MustParse(fmt.Sprintf("%dM", size))
 	}
 
+	var periodicity string
+	if c.LogCollector.Periodicity != "" {
+		if c.LogCollector.Periodicity == "1h" || c.LogCollector.Periodicity == "hourly" {
+			periodicity = "hourly"
+		} else if c.LogCollector.Periodicity == "weekly" || c.LogCollector.Periodicity == "monthly" {
+			periodicity = c.LogCollector.Periodicity
+		} else {
+			periodicity = "daily"
+		}
+	}
```

Comment:
can we do switch instead of if ifelse else like below for better readability?

```go
if c.LogCollector.Periodicity != "" {
    switch c.LogCollector.Periodicity {
    case "1h", "hourly":
        periodicity = "hourly"
    case "weekly", "monthly":
        periodicity = c.LogCollector.Periodicity
    default:
        periodicity = "daily"
    }
}
```